tools/RepoTasks/RepoTasks.Cmdlets/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="System.Management.Automation" version="6.1.7601.17515" targetFramework="net452" />
```

Comment:
You shouldn't need this the build system should be able to find system.management.automation from your reference assemblies